curl/readme.yaml

```diff
@@ -0,0 +1,18 @@
+name: curl
+description: |
+    Simple alpine based docker image with curl binary and SSL dependencies.
+
+    ## How to build
+
+    The Docker image can be built using the following command
+
+    ```{bash}
+    docker build -t <your Docker Hub ID>/curl .
+    ```
+
+    where the dot is the current folder, assuming you are running the docker command from the same folder of the Dockerfile.
+
+    ## License
```

Comment:
I don't think this section is necessary (it's covered by the `Third-Party Licenses` boilerplate that gets templated in)